parastell/invessel_build.py

```diff
@@ -540,14 +544,19 @@ class RadialBuild(object):
                         none is supplied, the 'component' key will be used.
                 }
             }.
+        separate_chamber (bool): if wall_s > 1.0, separate interior vacuum
```

Comment:
I don't think `chamber` is the right word here.  In no case is there a separate chamber.  Normally, the SOL and the plasma share a volume.  In fact, arguably, we never need to actually model the plasma separately from the SOL as they are both vacuum.

Maybe we call this `separate_sol` instead?

Reply:
"Separate" is meant as a verb here, not a noun. By this variable name, I mean that the vacuum chamber is being separated into the plasma and SOL if `True`, or kept as a single volume if `False`.

Reply:
Aha! I was definitely reading it as a verb.  Maybe `divide` is less ambiguous?

Reply:
Works for me!